example/tests/utils.py

```diff
@@ -1,21 +1,7 @@
 import json
 
-from django.utils.encoding import force_bytes, force_text
+from django.utils.encoding import force_text
 
 
 def load_json(data):
```

Comment:
Many of the code changes removed `load_json`. Is it still used somewhere or could it be deleted too?

Reply:
I would like to remove it as well. Unfortunalty there is one [test](https://github.com/django-json-api/django-rest-framework-json-api/blob/master/example/tests/integration/test_non_paginated_responses.py#L117) which still needs it.
Reason being is that it uses `RequestFactory` and not `APIClient`. In such a case response doesn't seem to provide a `.json()` function.

Reply:
Would it be possible to convert the test to use `APIClient` instead?

Reply:
Possibly. I have a look.

Reply:
I have found an ok way to convert it to `APIClient`. Now also removed `example/tests/utils.py`.